src/test/resources/sampleTemplates/properties.yml

```diff
@@ -0,0 +1,48 @@
+company: lm
+sbu: USCM
+org: Products
+engine: LMBDT
+rdsCredentialBrokerImage: "repo.forge.lmig.com/docker/grm/ah/ecs/rds-credential-broker:BUILD6-Rb63a8812e36fc465e855ca5066e753bf9fef3c0f"
```

Comment:
Wondering if we should keep LM specifics like a bunch of values in this file out of the GH repo

Reply:
Yep, absolutely - I'll make this generic instead of having LM values.